lib/resources/dstu2/condition.yaml

```diff
@@ -129,22 +129,35 @@ fields:
       "category": {
         "coding": [
           {
-            "system": "http://argonaut.hl7.org/extension-codes",
+            "system": "http://argonaut.hl7.org",
             "code": "problem",
             "display": "Problem"
           }
         ],
         "text": "Problem"
       }
     }
+    OR
+    {
```

Comment:
We have called it out as implementation note as a whole for the create and update functionality separately and also as a note on the binding as well on line 155.